pkcs11/util_pkcs11.c

```diff
@@ -1637,8 +1640,11 @@ yubihsm_pkcs11_object_desc *get_object_desc(yh_session *session,
   }
 
   object->object.type = type;
-
+#ifdef _MSVC
+  gettimeofday_win(&object->tv);
```

Comment:
why not use the earlier defined macro to get the right implementation no matter platform?